types/react-sizes/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for react-sizes 2.0
+// Project: https://github.com/renatorib/react-sizes#readme
+// Definitions by: {@janKir,@micahstubbs} <https://github.com/janKir> <https://github.com/micahstubbs>
```

Comment:
This line is what’s causing the builds to fail. It has to be in a particular format, because we parse the contributor in the header comment into some structured metadata. Here's an example from a random other package: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ac3f9f5cf90f47bf5af6683fc9e6cb8befe72d00/types/react-fa/index.d.ts#L3

Reply:
acknowledged, will fix the format 👌